android/src/main/java/com/twiliorn/library/CustomTwilioVideoView.java

```diff
@@ -523,10 +523,11 @@ public void connectToRoomWrapper(
                 // No need to connect to room if video creation failed
                 return;
             }
-        } else {
-            isVideoEnabled = false;
```

Comment:
The isVideoEnabled flag is used in onStart() when calling LocalVideoTrack.create() to determine if the new video track will be enabled initially.  So, setting it to false here will result in video capture being disabled on the first background/resume cycle.

In trying to understand why they were doing that here, my best guess is that they just wanted it set to false if the "enableVideo" param was false in the call here to connectToRoomWrapper() (which is invoked from TwilioAdapter::connect()).  But they accidentally lumped it in with a check that also will set it to false if there was already a cameraCapturer created before this, which seems incorrect.

In our use case, the cameraCapturer is populated already before we get here because we call TwilioAdapter::cameraOn() in the RoomAbstraction setup (when video is enabled from the lobby).  That in turn results in calling into this library to toggleVideo() which calls createLocalVideo(), where the cameraCapturer is created.

As far as I can tell, calling toggleVideo() before connect() seems like a valid thing to do, so there's no reason to set isVideoEnabled to false just because the cameraCapturer was already created before we got here.  I believe what they really intended was just to set isVideoEnabled to whatever value of enableVideo was passed in.